test/ergw_http_api_SUITE.erl

```diff
@@ -262,11 +394,32 @@ http_api_prometheus_metrics_req(_Config) ->
 http_api_delete_sessions() ->
     [{doc, "Check DELETE /contexts/count API"}].
 http_api_delete_sessions(Config) ->
-    Res1 = json_http_request(delete, "/contexts/0"),
-    ?match(#{<<"contexts">> := 0}, Res1),
+    {GtpC0, _, _} = create_pdp_context(Config),
+    {_, _, _} = create_pdp_context(random, GtpC0),
+    ContextsCount = length(ergw_api:contexts(all)),
+    ?match(2, ContextsCount),
+    Res = json_http_request(delete, "/api/v1/contexts/2"),
+    ct:sleep(80000),
```

Comment:
80 seconds is too high, wait4tunnels/1 with a reasonable timeout (the other tests use 2 seconds) should work.

Reply:
at the moment this is the optimal amount of time, there was a check with 60 seconds and looks like contexts is not remove at this time, I tried to find out the reason for such a delay, but unfortunately I could not find the reasons about it... Maybe you can help me understand why we have here such a delay?

Reply:
the API call triggers a delete bearer request from the PGW. If you don't respond to that requests, it will be resent a couple of times. This in the end lead to 60 seconds timeout.
You have to make you actually response, check for a sample check delete_bearer_requests_multi  in the PGW suite and delete_pdp_context_requested in the GGSN suite.

Reply:
Thanks a lot. Fixed.